BuildScript/BuildTargets/Testing.targets

```diff
@@ -128,31 +124,37 @@
     <!-- Run tests -->
 
     <PropertyGroup>
-      <_testName>$(_testAssemblyFileName).$(ConfigurationID).$(_platform)</_testName>
-      <_testName Condition="'$(_databaseSystem)' != '$(NoDatabaseRequired)'">$(_testName).$(_databaseSystem)</_testName>
-      <_testName Condition="'$(_browser)' != '$(NoBrowserRequired)'">$(_testName).$(_browser)</_testName>
-      <_testResultFile>$(LogDirectory)$(_testName).xml</_testResultFile>
+      <_mergedTestCategoriesToExclude>$(TestCategoriesToExclude.Replace(';', ',')),$(_excludeCategoriesInTestConfiguration)</_mergedTestCategoriesToExclude>
```

Comment:
Question: should we unify the two statements to append the Trim to this line instead of having it separately? Of course, it's possible that this won't work without escessive string operations making the code unreadable. -> try and see

Reply:
I see your point and tried to achieve the exact same thing you suggest, but honestly, it took a lot of playing around with Replace, Trim, Join etc. reach a solution that is fairly readable and only 2 lines of code, so I would leave it at that if that's alright